### PR TITLE
Fix config file

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *Some patterns were not replaced during the install/upgrade*

## Solution
- *Fix that*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/bitwarden_ynh%20PR1%20(yalh76)/)  
Revision: 43562ced2f1c0a67036f7904d2d36c1d65dfc87c 
image

### CODE REVIEW

**Comment by kay0u** [comment]
I still can't login to bitwarden with an user account... (I create a new user, with an invitation etc). I got this error in my browser:

> ERROR Error: "Uncaught (in promise): Error: Token not found.
[...]
Erreur dans les liens source : SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data

File: scripts/install [hunk 1]
```diff
@@ -150,7 +150,7 @@ ynh_replace_string --match_string="__ADMIN_TOKEN__" --replace_string="$admin_tok
 ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain" --target_file="$config"
 ynh_replace_string --match_string="__WEBSOCKET_PORT__" --replace_string="$websocket_port" --target_file="$config"
 ynh_replace_string --match_string="__ROCKET_PORT__" --replace_string="$rocket_port" --target_file="$config"
-ynh_replace_string --match_string="__APP__" --replace_string="$rocket_port" --target_file="$config"
+ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file="$config"
```
Comment (kay0u) [inline scripts/install]: Indeed, shitty copy/pasta

Comment (yalh76) [reply scripts/install]: Yes :) corrected and tested: 
1. Install
2. Go to the mails to retrieve admin token
3. Log to the admin interface
4. send an invitation
5. Go to the mail, click on the invitation link
6. create an account
7. Login as user, logout, loginback

Comment (kay0u) [reply scripts/install]: 1. Install - OK
2. Go to the mails to retrieve admin token - OK
3. Log to the admin interface - OK
4. send an invitation - OK
5. Go to the mail, click on the invitation link - OK
6. create an account - OK
7. Login as user, logout, loginback - KO with what I log above

I'm on ARM board btw, could explain why it failed

Comment (yalh76) [reply scripts/install]: Seems weird, as even arm-ci is able to install bitwarden_ynh: https://ci-apps-arm.yunohost.org/ci/job/740

Comment (kay0u) [reply scripts/install]: Install is ok, login not --'

Comment (yalh76) [reply scripts/install]: When you do `journalctl -u bitwarden -fe`, you see something related to the failed login ?

Comment (yalh76) [reply scripts/install]: I will launch a test on AARCH64 VM, see you in 5 hours ^^

Comment (yalh76) [reply scripts/install]: Test is ok on a AARCH64 VM ....
If you have more logs, it would be helpfull

**Pull request merged**